integration_tests/tests/vault/set_admin.rs

```diff
@@ -0,0 +1,42 @@
+#[cfg(test)]
+mod tests {
+    use jito_vault_core::config::Config;
+    use solana_sdk::signature::{Keypair, Signer};
+
+    use crate::fixtures::{fixture::TestBuilder, vault_client::VaultRoot};
+
+    #[tokio::test]
+    async fn test_set_admin() {
```

Comment:
can you add test that tries to set admin with bad admin?

Reply:
Yep!

Reply:
Done